flake.nix

```diff
@@ -25,7 +25,7 @@
           }: {
             devShells.default = pkgs.mkShell {
               inputsFrom = [ inputs.holochain.devShells.${system}.holonix ];
-              packages = [ pkgs.nodejs-18_x ];
+              packages = [ pkgs.nodejs-18_x pkgs.sqlite ];
```

Comment:
this isn't required anymore since holochain was fixed to use bundled sqlite in all circumstnaces

Reply:
Yeah okey... In which version? Should we upgrade holochain in the main branch of the scaffolding tool?

Reply:
here's a PR for that: #97